src/service.ts

```diff
@@ -207,36 +209,46 @@ export class DebugService implements IDebugger {
     return reply.body.variables;
   };
 
-  getBreakpoints = (): DebugProtocol.SourceBreakpoint[] => {
-    return this._model.breakpointsModel.breakpoints.map(breakpoint => {
-      return {
-        line: breakpoint.line
-      };
-    });
-  };
-
   setBreakpoints = async (
     breakpoints: DebugProtocol.SourceBreakpoint[],
```

Comment:
Do we need to keep this method? We could move its implementation into `updateBreakpoints` since you are always supposed to send a `dumpCell` request before a `setBreakpoints` request.

Reply:
Actually we may need to keep it fo the case when the code changes in a cell: we may want to remove the breakpoints from the backend, in this case we already have the hash for the file name.

Reply:
Yes we can keep it around for a while, but happy to move it to `updateBreakpoints` if it is not needed anymore.